### PR TITLE
Task/dockerfile deployment script

### PR DESCRIPTION
**Pull request by pundlikintel**
1. Read optional input for environment file. use /tmp/execute_workload_flow.env
2. Read, validate and export environment variables from environment file.
3. Change kbs url.
4. Decode and print attestation_token after successful call.
5. Using constant for code.

### CODE REVIEW

File: deployment/sample-workload/execute_workload_flow.sh [hunk 1]
```diff
@@ -33,6 +74,40 @@ function makeHttpsCall(){
...
+    fi
+    printf "Using $envFileName environment file\n"
+    checkIfFileExists $envFileName
+    if [ $? == 1 ]; then
+        echo -e "\u274c  Environment file ($envFileName) does not exist, exiting... Please create env file and rerun script"
+        return 1
+    fi
+    export $(grep -v '^#' $envFileName | xargs -d '\n')
+    
+    #check if environment file exists
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: misplaced comment.

File: deployment/sample-workload/execute_workload_flow.sh [hunk 2]
```diff
@@ -33,6 +74,40 @@ function makeHttpsCall(){
...
+    if [[ -z "$KBS_URL" ]];then
+        printf "${CODE_ERROR}%s${CODE_NC}" "KBS_URL must be present"
+        hasError=true
+    fi  
+    if [[ -z $KBS_KEY_ID ]];then
+        printf "${CODE_ERROR}%s${CODE_NC}" "KBS_KEY_ID must be present"
+        hasError=true
+    fi  
+    if [[ $hasError == true ]];then
+        printf "For environment variable refer "'\e]8;;%s\e\\%s\e]8;;\e\\\n' "${HELP_LINK}" "${HELP_LINK}"
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: How about `For info on environment variable refer...`?

File: deployment/sample-workload/execute_workload_flow.sh [hunk 3]
```diff
@@ -1,5 +1,39 @@
 #!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# Script is used for executing workload flow
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh]: duplicate comment.

File: deployment/sample-workload/execute_workload_flow.sh [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+readonly LABEL_DECRYPT_KEY_DESC="Decrypting Model using encryption key from KBS."
+
+readonly LABEL_EXE_MODEL="Execute model"
+readonly LABEL_EXE_MODEL_DESC="Executing model with test data."
+
+readonly LABEL_RST_MODEL="Reset Model"
+readonly LABEL_RST_MODEL_DESC="Clearing decrypted model from memory."
+
+#link
+readonly HELP_LINK="https://github.com/arvind5/trustauthority-samples/blob/main/tdx-ml-workload/README.md"
```
Comment (arvind5) [inline deployment/sample-workload/execute_workload_flow.sh:34]: update the correct link.

**Review by arvind5 — CHANGES_REQUESTED** [review]

File: deployment/sample-workload/execute_workload_flow.sh (hunk 4 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh:34]: Removed link

File: deployment/sample-workload/execute_workload_flow.sh (hunk 3 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Removed

File: deployment/sample-workload/execute_workload_flow.sh (hunk 2 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Removed line

File: deployment/sample-workload/execute_workload_flow.sh (hunk 1 above)
Comment (pundlikintel) [reply deployment/sample-workload/execute_workload_flow.sh]: Replaced at correct place

**Review by arvind5 — APPROVED** [review]

**Pull request merged**